Spigot-API-Patches/0039-Added-source-to-PlayerExpChangeEvent.patch

```diff
@@ -0,0 +1,57 @@
+From 6f49e1bf0a9b07ddef4f7232fd0f5a97c6545658 Mon Sep 17 00:00:00 2001
+From: AlphaBlend <whizkid3000@hotmail.com>
+Date: Thu, 8 Sep 2016 08:47:08 -0700
+Subject: [PATCH] Added source to PlayerExpChangeEvent
+
+
+diff --git a/src/main/java/org/bukkit/event/player/PlayerExpChangeEvent.java b/src/main/java/org/bukkit/event/player/PlayerExpChangeEvent.java
+index f37491d..e856ac8 100644
+--- a/src/main/java/org/bukkit/event/player/PlayerExpChangeEvent.java
++++ b/src/main/java/org/bukkit/event/player/PlayerExpChangeEvent.java
+@@ -1,5 +1,6 @@
+ package org.bukkit.event.player;
+ 
++import org.bukkit.entity.Entity; // Paper
+ import org.bukkit.entity.Player;
+ import org.bukkit.event.HandlerList;
+ 
+@@ -8,14 +9,36 @@ import org.bukkit.event.HandlerList;
+  */
+ public class PlayerExpChangeEvent extends PlayerEvent {
+     private static final HandlerList handlers = new HandlerList();
++    // Paper start
++    private Entity source;
+     private int exp;
+ 
+     public PlayerExpChangeEvent(final Player player, final int expAmount) {
```

Comment:
This constructor never had documentation to begin with (and wasn't the constructor I added to begin with). In fact, most if not all of bukkit event constructors don't have javadocs except the ones that are deprecated, in which case they will include redirect notices.

So I didn't feel the need, and neither of the 2 official developers who commented saw reason to add a comment.